test/api/APIAOTCoreTest.cpp

```diff
@@ -29,14 +29,11 @@
 #include <utility>
 #include <vector>
 
-#if defined(linux) || defined(__linux) || defined(__linux__) ||                \
-    defined(__gnu_linux__)
+#if WASMEDGE_OS_LINUX
```

Comment:
Do you need to include any header related to these macro variables?

Reply:
this identifier is defined in `common/defines.h`